packages/cyberstorm/src/components/Button/Button.tsx

```diff
@@ -90,6 +92,7 @@ const Button = React.forwardRef<
   if (plain) {
     const fRef = forwardedRef as React.ForwardedRef<HTMLDivElement>;
     const ref = fRef || fallbackRef;
+    // TODO: Add disabled mode
```

Comment:
Doesn't this commit already add the disabled mode?

Reply:
The plain button needs more than adding the disabled attribute to the element, since it's a div.